src/installer/pkg/sfx/installers/dotnet-hostfxr.proj

```diff
@@ -36,7 +36,7 @@
   <Target Name="AddLinuxPackageInformation" BeforeTargets="GetDebInstallerJsonProperties;GetRpmInstallerJsonProperties">
     <ItemGroup Condition="'$(GenerateDeb)' == 'true'">
       <LinuxPackageDependency Include="libc6;libgcc1;libstdc++6" />
-      <LinuxPackageDependency Include="dotnet-host" Version="$(Version)" />
+      <LinuxPackageDependency Include="dotnet-host" Version="$(InstallerPackageVersion)" />
```

Comment:
why not `HostPackageVersion` ?

Reply:
> why not `HostPackageVersion` ?

Yeah, I agree - `InstallerPackageVersion` seems to be defined for Windows build only, while `HostPackageVersion` is available everywhere.

Reply:
InstallerPackageVersion is produced by the installer targets and has the Debian version format when building debs (6.0.0~dev instead of 6.0.0-dev). The fact that we are using a property that used 6.0.0-dev is causing uptake issues into dotnet/installer.

I’m not using a local definition of the InstallerPackageVersion property.